site/_posts/2019-08-12-announce_j_name.html.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Name of J Release
+author: jprause
+date: 2019-08-12
+tags: releases announcements
+comments: true
+published: true
+---
+
+The results of the poll on the name of the J release are in. **Vlastimail Jansa** is the winner! Looking forward to all the striking features that will come in the **Jansa** release. Thanks to all that voted - we had 69 total votes for the J-Release.
```

Comment:
First name should be Vlastimil not Vlastimail (remove extra `a`).

And maybe changing the adjective "striking" to "exciting" or something else. I think we used striking to describe the *Hammer* release :)